resources/options.html

```diff
@@ -52,7 +52,7 @@
                 <p style="font-family: Roboto,sans-serif; font-size:11pt">
                 The following data can be tracked from me, while I'm reviewing pull requests on GitHub:
                 </p>
-                <ul id="privacy_sub">
+                <ul id="privacySub">
```

Comment:
The 'Sub' parts in these IDs indicate that it covers all suboptions of a card.
Coming up with a better name would be nice.. :P Any suggestions? :innocent: